### PR TITLE
Add build process to create geoserver with additional community plugin

### PR DESCRIPTION
**Pull request by EthanGrahn**
Created a custom dockerfile and added a new file that Jenkins will use to build and push the new image to Artifactory.

### CODE REVIEW

**Comment by EthanGrahn** [comment]
@kkehl-usgs could you verify that this implementation is valid?

**Comment by kkehl-usgs** [comment]
This implementation looks valid as far as I can tell, but I don't have experience with changing the artifactory or image name.

@ayan-usgs may have an opinion.

As an aside, I wonder if we can move away from this ecs-pipeline.  In the case of the iow-geoserver, I think it could be deployed much more easily using serverless. See this example:   https://github.com/svdgraaf/serverless-fargate-tasks.   I only started looking at this today, not sure how to configure the max number of tasks, but still.   It would be less confusing!

**Comment by EthanGrahn** [comment]
I agree that it should be moved away from the pipeline. Maybe we can use this as a proof of concept that building with the extension works, then we work on that transition.

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,9 @@
+FROM kartoza/geoserver:2.18.0
+
+USER root
+
+RUN wget https://build.geoserver.org/geoserver/2.18.x/community-latest/geoserver-2.18-SNAPSHOT-s3-geotiff-plugin.zip && \
+    unzip -q -n geoserver-2.18-SNAPSHOT-s3-geotiff-plugin.zip -d /usr/local/tomcat/webapps/geoserver/WEB-INF/lib/ && \
+    rm geoserver-2.18-SNAPSHOT-s3-geotiff-plugin.zip
+
+USER geoserveruser
```
Comment (ayan-usgs) [inline Dockerfile:9]: Does this user already exist within the container? If not, you'll have to create it.

Comment (EthanGrahn) [reply Dockerfile:9]: Yes, it's part of the base image. https://hub.docker.com/r/kartoza/geoserver/dockerfile

**Pull request merged**